Lib9c.Policy/NCStagePolicy.cs

```diff
@@ -59,7 +59,18 @@ public IEnumerable<Transaction> Iterate(BlockChain blockChain, bool filtered = t
                     }
 
                     s.Add(tx);
-                    if (s.Count > _quotaPerSigner)
+                    int txQuotaPerSigner = _quotaPerSigner;
+                    if (_accessControlService != null)
+                    {
+                        // update txQuotaPerSigner if ACS returns a value for the signer.
+                        int? acsTxQuota = _accessControlService.GetTxQuota(tx.Signer);
+                        if (acsTxQuota.HasValue)
+                        {
+                            txQuotaPerSigner = (int)acsTxQuota;
+                        }
```

Comment:
Not critical but in C# way:

```suggestion
                        if (_accessControlService.GetTxQuota(tx.Signer) is { } acsTxQuota)
                        {
                            txQuotaPerSigner = acsTxQuota;
                        }
```